binaries/scootapi/main.go

```diff
@@ -11,7 +11,7 @@ func main() {
 	protocolFactory := thrift.NewTBinaryProtocolFactoryDefault()
 	transportFactory := thrift.NewTTransportFactory()
 
-	dialer := client.NewDialer("localhost:9090", transportFactory, protocolFactory)
+	dialer := client.NewDialer(transportFactory, protocolFactory)
```

Comment:
I'm not sure that removing addr is necessary here, just pass a blank string like in cli.go.